### ISSUE TITLE
Allow users to query based on Edge seeds

### ISSUE THREAD

**Issue by p013570** [gchq/gaffer-tools#389]

**Comment by p013570** [comment]
Linked to #398

**Comment by n3101** [comment]
The UI fails to meet our Accessibility policy. It is unmaintainable and will be removed shortly.
